realm/src/androidTest/java/io/realm/SortTest.java

```diff
@@ -0,0 +1,202 @@
+/*
```

Comment:
All other files that test a class is called CLASSTest.java, so when we test cross class functionality, maybe we should use another naming practice.